salt/modules/virt.py

```diff
@@ -683,21 +684,24 @@ def _gen_pool_xml(name,
                   source_hosts=None,
                   source_auth=None,
                   source_name=None,
-                  source_format=None):
+                  source_format=None,
+                  source_initiator=None):
     '''
     Generate the XML string to define a libvirt storage pool
     '''
     hosts = [host.split(':') for host in source_hosts or []]
     source = None
-    if source_devices or source_dir or source_adapter or hosts or source_auth or source_name or source_format:
+    if source_devices or source_dir or source_adapter or hosts or source_auth or source_name or source_format \
+            or source_initiator:
```

Comment:
Can I suggest to use `any()`?

Reply:
Thanks for the tip, I didn't know `any()`. I'll update the upstream PR and this will cascaded a little later here